### PR TITLE
Blacklist headline

### PR DESCRIPTION
**Pull request by dlukeomalley**

### CODE REVIEW

File: src/content/headsup/OverrideHeadsup.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  private renderIndicator(overrideType: OverrideType) {
+    switch (overrideType) {
+      case "blacklist":
+        return <CrossIndicator />;
+      case "whitelist":
+        return <CheckIndicator />;
+      case "promote":
+        return <PromoteIndicator />;
+      default:
+        return <CircleIndicator />;
```
Comment (dlukeomalley) [inline src/content/headsup/OverrideHeadsup.tsx]: This feels like a stronger error / gray / unknown state on our end. Thoughts on displaying this as a grey info sign or something related?

CC @tnarla - how do we account for "error" states like this in the current mocks?

File: src/content/headsup/OverrideHeadsup.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    const { override, children } = this.props;
+    const { showMore } = this.state;
+
+    return (
+      <>
+        <OverrideHeadsup
+          override={override}
+          onReportClick={
+            l(
+              "report-override-click",
```
Comment (dlukeomalley) [inline src/content/headsup/OverrideHeadsup.tsx]: What do you think about `preflight-report-mistake-click`?

@tnarla Does this follow the conventions you've been working on?

File: src/content/headsup/OverrideHeadsup.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        />
+        {showMore && children}
+      </>
+    );
+  }
+
+  private handleReportClick: React.MouseEventHandler = () => {
+    MainToaster.show({
+      message:
+        "Thanks for letting us know. We'll take a look and make it right.",
```
Comment (dlukeomalley) [inline src/content/headsup/OverrideHeadsup.tsx]: What do you think about asking for a more in depth report from people by opening an email template or have it open/file an issue against secarta-extension?

File: src/content/headsup/OverrideHeadsup.tsx (hunk 2 above)
Comment (tnarla) [reply src/content/headsup/OverrideHeadsup.tsx]: Hey! so the convention I was thinking was ('product-name'-'feature'-'element'-'action') so in this case what Luke wrote should be good!

File: src/content/headsup/OverrideHeadsup.tsx (hunk 1 above)
Comment (tnarla) [reply src/content/headsup/OverrideHeadsup.tsx]: I can mock up a grey state for if data is missing. The icon could maybe be a question mark or something indicating 'missing' or 'unknown'.

**Pull request merged**